grammars/php.cson

```diff
@@ -721,9 +721,9 @@
         ]
       }
       {
-        'begin': '(?i)^\\s*(namespace)\\b\\s+(?=([a-z0-9_\\\\]+\\s*($|[;{]|(\\/[\\/*])))|$)'
+        'begin': '(?i)(^\\s*|\\s)(namespace)\\b\\s+(?=([a-z0-9_\\\\]+\\s*($|[;{]|(\\/[\\/*])))|$)'
```

Comment:
Changed to `\\s*` but it did work with `\\s` also. Fixed so it's a non-capturing group.